.github/workflows/ci.yml

```diff
@@ -28,21 +31,42 @@ jobs:
           else
             GIT_REF="${{ github.ref }}"
           fi
-          echo "GIT_REF=${GIT_REF}" >>"$GITHUB_ENV"
+          echo "git_ref=${GIT_REF}" >>"$GITHUB_OUTPUT"
+
+  docker_smoketests:
+    name: Smoketests
+    needs: find_ref
+    strategy:
+      matrix:
+        include:
+          - { runner: spacetimedb-runner, smoketest_args: --docker }
+          - { runner: windows-latest, smoketest_args: --no-build-cli }
+        runner: [spacetimedb-runner, windows-latest]
+    runs-on: ${{ matrix.runner }}
+    steps:
       - name: Checkout sources
         uses: actions/checkout@v4
         with:
-          ref: ${{ env.GIT_REF }}
+          ref: ${{ needs.find_ref.outputs.git_ref }}
       - uses: dsherret/rust-toolchain-file@v1
-      - uses: actions/setup-dotnet@v3
+      - uses: actions/setup-dotnet@v4
         with:
           global-json-file: modules/global.json
       - name: Start containers
+        if: runner.os == 'Linux'
         run: docker compose up -d
+      - name: Start database
+        if: runner.os == 'Windows'
+        run: |
+          cargo build -p spacetimedb-cli -p spacetimedb-standalone -p spacetimedb-update
+          Start-Process target/debug/spacetimedb-cli.exe start
+          cd modules
+          dotnet workload config --update-mode workload-set
+          dotnet workload update
```

Comment:
dumb question - do you know why this dotnet-juggling is only needed on windows?

Reply:
No - I think GitHub's runner environment is just messed up.

Reply:
Can you just add a comment saying as much? just a "we're not sure why this is needed on windows, but it made the thing work" type of thing so that future maintainers aren't terrified of changing it.